_docs/_latest/concepts/index.md

```diff
@@ -29,3 +29,8 @@ complex security policies.
 
 Learn about the types of service accounts you can use with Forseti and the roles associated with
 using them.
+
+**[Forseti Real Time Enforcer]({% link _docs/latest/concepts/real-time-enforcer.md %})**
```

Comment:
nit: please add a hypen for real time `Real-Time Enforcer`.  Consistent with other services.

Reply:
Updated.